src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,72 @@
 package gr11review.part2;
 
+import java.io.*;
+import java.io.FileNotFoundException;
+
 public class Utility {
-    
-}
\ No newline at end of file
+  public static String zipZap(String strWord){
```

Comment:
All methods should have a javadocs comment header that describes the purpose, parameters, and return value.